charts/ceph-csi-cephfs/templates/nodeplugin-clusterrole.yaml

```diff
@@ -14,4 +14,10 @@ rules:
   - apiGroups: [""]
     resources: ["nodes"]
     verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "list"]
```

Comment:
just the `get` action would be sufficient I believe.

Reply:
Thanks for noticing, you are correct. I will fix it!